index.js

```diff
@@ -0,0 +1,28 @@
+// !! IMPORTANT README:
+
+// You may add additional external JS and CSS as needed to complete the project, however the current external resource MUST remain in place for the tests to work. BABEL must also be left in place. 
+
+/***********
+INSTRUCTIONS:
+  - Select the project you would 
+    like to complete from the dropdown 
+    menu.
+  - Click the "RUN TESTS" button to
+    run the tests against the blank 
+    pen.
+  - Click the "TESTS" button to see 
+    the individual test cases. 
+    (should all be failing at first)
+  - Start coding! As you fulfill each
+    test case, you will see them go   
+    from red to green.
+  - As you start to build out your 
+    project, when tests are failing, 
+    you should get helpful errors 
+    along the way!
+    ************/
+
+// PLEASE NOTE: Adding global style rules using the * selector, or by adding rules to body {..} or html {..}, or to all elements within body or html, i.e. h1 {..}, has the potential to pollute the test suite's CSS. Try adding: * { color: red }, for a quick example!
+
+// Once you have read the above messages, you can delete all comments. 
+"use strict";
```

Comment:
Required Changes:
- [ ] This was a great review @sheylaPozo. I noticed a great use of more than one comments in the review. For the first change, you did try to offer an explanation of why the change was required. The second requirement seemed to have lacked an explanation. Please try to always include an explanation, a "why", for each requested change. it makes the review much more complete. 
- [ ] Another fundamental aspect of this project will be to make the review as much humanly as possible. I think you ought to use a couple of emojis and/or gifs in the review. They will make the code review more human-friendly :alien::robot:
- [ ] As for this project we need to ensure we have grasped the proper use of the `GitHub inline comments feature` at least once to leave review comments. Make sure to leave the comment under a line or block of code that fits the context the comment is about.
- [ ] Kindly make these few changes then submit them for a review.